src/FamilyService.Validation/Child/CreateChildRequestValidator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using FluentValidation;
+using LT.DigitalOffice.FamilyService.Data.Interfaces;
+using LT.DigitalOffice.FamilyService.Models.Dto.Requests.Child;
+using LT.DigitalOffice.FamilyService.Validation.Child.Interfaces;
+
+namespace LT.DigitalOffice.FamilyService.Validation.Child
+{
+  public class CreateChildRequestValidator : AbstractValidator<CreateChildRequest>, ICreateChildRequestValidator
+  {
+    private static Regex NameRegex = new(@"^([a-zA-Zа-яА-ЯёЁ]+|[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+|[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+)$");
+
+    public CreateChildRequestValidator(
+      IChildRepository _userRepository)
+    {
+      RuleFor(r => r.Name)
+        .Cascade(CascadeMode.Stop)
+        .MinimumLength(1).WithMessage("Minimum lenght is 1.")
```

Comment:
use NotEmpty

when you use the minimum length, the validator will not catch the string with white spase